src/main/java/org/broadinstitute/hellbender/tools/copynumber/CallCopyRatioSegments.java

```diff
@@ -75,7 +76,7 @@
     public static final String NEUTRAL_SEGMENT_COPY_RATIO_UPPER_BOUND_LONG_NAME = "neutral-segment-copy-ratio-upper-bound";
     public static final String OUTLIER_NEUTRAL_SEGMENT_COPY_RATIO_Z_SCORE_THRESHOLD_LONG_NAME = "outlier-neutral-segment-copy-ratio-z-score-threshold";
     public static final String CALLING_COPY_RATIO_Z_SCORE_THRESHOLD_LONG_NAME = "calling-copy-ratio-z-score-threshold";
-
+    public static final String IGV_COMPATIBLE_FILE_SUFFIX = ".igv.seg";
```

Comment:
This should be called `LEGACY_SEGMENTS_FILE_SUFFIX` for consistency with the analogous constants in `ModelSegments` (alternatively, the names of those constants could be changed).

Reply:
Done